sqlparser/seata-sqlparser-druid/src/main/java/io/seata/sqlparser/druid/SQLOperateRecognizerHolderFactory.java

```diff
@@ -39,7 +39,7 @@ public static SQLOperateRecognizerHolder getSQLRecognizerHolder(String dbType) {
         if (RECOGNIZER_HOLDER_MAP.get(dbType) != null) {
             return RECOGNIZER_HOLDER_MAP.get(dbType);
         }
-        SQLOperateRecognizerHolder recognizerHolder = EnhancedServiceLoader.load(SQLOperateRecognizerHolder.class, dbType, SQLOperateRecognizerHolderFactory.class.getClassLoader());
+        SQLOperateRecognizerHolder recognizerHolder = EnhancedServiceLoader.load(SQLOperateRecognizerHolder.class, dbType);
```

Comment:
how to process class isolation?

Reply:
It is true that the isolated class loader should be used here. However, there are some problems with the DruidIsolationClassLoader when loading resources (no distinction is made between internal resources, resulting in the SPI Implementations that in sqlparser.jar will be loaded twice). I will try to fix this problem recently.

Reply:
Now rewrite getResource() to avoid repeating load of the DRUID resources. PTAL.